tests/test_launcher.py

```diff
@@ -583,3 +558,95 @@ def test_docker_compose(monkeypatch):
     )
     solver.file.read_case(file_name=case_file_name)
     solver.exit()
+
+
+def test_respect_driver_is_not_null():
+    assert (
+        _build_fluent_launch_args_string(
+            ui_mode=UIMode.GUI,
+            graphics_driver=(
+                FluentWindowsGraphicsDriver.DX11
+                if is_windows()
```

Comment:
I'm confused here. We should be controlling the behaviour of `is_windows()` within the test so that we deliberately test each platform-specific behaviour. The outcome of test shouldn't depend on the actual, real platform type. Perhaps I'm misunderstanding?

Reply:
Actually I referred existing tests and added tests based on this, otherwise I had added separate tests for both Windows and Linux. We will update these tests. Thank you.